pact/coin-contract/v2/coin.pact

```diff
@@ -326,6 +330,20 @@
       (credit account account-guard amount))
     )
 
+  (defun slash:string (account:string amount:decimal)
+    @doc "Internal function for the termination of coins. This function \
+         \cannot be used outside of the coin contract."
+
+    @model [ (property (valid-account account)) ]
```

Comment:
Both this and `coinbase` should have more properties, namely amount > 0, `valid-account`.

Also, `transfer-create` is missing a lot of properties that `transfer` has.